src/Lighthouse/LighthouseHostFactory.cs

```diff
@@ -24,13 +24,11 @@ public static class LighthouseHostFactory
         {
             systemName = systemName ?? Environment.GetEnvironmentVariable("ACTORSYSTEM")?.Trim();
 
-
-            // Set environment variables for use inside Akka.Bootstrap.Docker
-            // If overrides were provided to this method.
-            //if (!string.IsNullOrEmpty(ipAddress)) Environment.SetEnvironmentVariable("CLUSTER_IP", ipAddress);
-
-            //if (specifiedPort != null)
-            //    Environment.SetEnvironmentVariable("CLUSTER_PORT", specifiedPort.Value.ToString());
+            var argConfig = "";
+            if (ipAddress != null)
+                argConfig += $"akka.remote.dot-netty.tcp.public-hostname = {ipAddress}\n";
+            if (specifiedPort != null)
+                argConfig += $"akka.remote.dot-netty.tcp.port = {specifiedPort}";

```

Comment:
Is this reliable? What about this:
```csharp 
if (specifiedPort.HasValue)
                argConfig += $"akka.remote.dot-netty.tcp.port = {specifiedPort.Value}";
```

Reply:
Its basically the same, its a nullable, it can be directly compared to a null